src/matching_market.sol

```diff
@@ -207,23 +220,24 @@ contract MatchingMarket is MatchingEvents, ExpiringMarket, DSNote {
         return true;
     }
 
-    //set the minimum sell amount for a token
+    //set the minimum sell amount for a token. Uses Uniswap as a price oracle.
     //    Function is used to avoid "dust offers" that have
     //    very small amount of tokens to sell, and it would
     //    cost more gas to accept the offer, than the value
     //    of tokens received.
     function setMinSell(
-        ERC20 pay_gem,     //token to assign minimum sell amount to
-        uint dust          //maker (ask) minimum sell amount
+        ERC20 pay_gem     //token to assign minimum sell amount to
     )
         public
-        auth
         note
         returns (bool)
     {
-        _dust[address(pay_gem)] = dust;
-        emit LogMinSell(address(pay_gem), dust);
-        return true;
+        require(msg.sender == tx.origin, "No indirect calls please");
```

Comment:
We might want to remove the `note` event in this function as `_setMinSell` has its own custom event log.